flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -110,7 +110,12 @@
 	public static final String EXECUTION_RETRY_DELAY_KEY = "execution-retries.delay";
 	
 	// -------------------------------- Runtime -------------------------------
-	
+
+	/**
+	 * JVM heap size (in megabytes) for the JobManager
+	 */
+	public static final String JOB_MANAGER_HEAP_MEMORY_KEY = "jobmanager.heap.mb";
```

Comment:
Can we introduce these values as a `ConfigOption` instead of an entry to `ConfigConstants`? I think it might go to `TaskManagerOptions` and to `JobManagerOptions`.